<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Race Results</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
    <item row="2" column="0">
     <widget class="QLabel" name="label_lastupdate">
      <property name="text">
       <string>Last Update:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="button_forceupdate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Force Update</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_event">
      <property name="text">
       <string>Event:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_org">
      <property name="text">
       <string>Organization:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="text_org">
      <property name="text">
       <string>orgName</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="button_service">
      <property name="text">
       <string>Start Service</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="text_event">
      <property name="text">
       <string>event#</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="text_lastupdate">
      <property name="text">
       <string>timestamp</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="button_config">
      <property name="text">
       <string>Configuration</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_config</sender>
   <signal>clicked()</signal>
   <receiver>main_window</receiver>
   <slot>modify_config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_forceupdate</sender>
   <signal>clicked()</signal>
   <receiver>main_window</receiver>
   <slot>force_update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_service</sender>
   <signal>clicked()</signal>
   <receiver>main_window</receiver>
   <slot>toggle_service()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modify_config()</slot>
  <slot>toggle_service()</slot>
  <slot>force_update()</slot>
 </slots>
</ui>
